cellmaps_generate_hierarchy/runner.py

```diff
@@ -105,7 +106,7 @@ def _register_computation(self, generated_dataset_ids=[]):
         :return:
         """
         logger.debug('Getting id of input rocrate')
-        input_dataset_id = self._provenance_utils.get_id_of_rocrate(self._inputdir)
+        input_dataset_id = self._provenance_utils.get_id_of_rocrate(self._inputdirs)
```

Comment:
get_id_of_rocrate can't take multiple paths as input yet (maybe there is a pull request that includes this feature. Should this be switched to a for loop and input_dataset_id switched to list?

Reply:
going to merge and fix after merge